process/release_process.md

```diff
@@ -0,0 +1,155 @@
+
+In our current practical scenario, we have an Edge Orchestrator reference project that consists of the following high-level components:
+
+- **xt-core** - a set of repositories inside the xen-troops GitHub organization that contains the core build functionality mainly as Yocto layers.
+  Note! It is up to xen-troops maintainers to define which repositories will be included in the xt-core abstraction. Currently, the most obvious candidates are listed below:
+  https://github.com/xen-troops/meta-xt-common
+  https://github.com/xen-troops/meta-xt-rcar
+  https://github.com/xen-troops/meta-xt-vhost
+  etc.
+- **AosEdge aos-rcar-gen4** - https://github.com/aoscloud/meta-aos-rcar-gen4. Moulin configuration and Yocto layers for R-Car Gen4 AosEdge distro.
+- **Cockpit meta-xt-prod-cockpit-rcar** - https://github.com/xen-troops/meta-xt-prod-cockpit-rcar. Reference demo product that implements the Cockpit multimedia system.
+
+In other words, we have **xt-core** base functionality—a reusable huge SW component consisting of multiple parts and moulin builds that want to create their releases relying on the released versions of the xt-core. Making the release means different things for these two types of entities. Let's describe the release process in these two cases.
+
+## How to Create a Release?
+
+### xt-core
+
+- For the xt-core the evolutionary development is used in the **'main'** branch.
+- Each xt-core repository's **'main'** branch uses [semantic versioning](https://semver.org) to create release tags, such as v1.1.1, v2.2.2, etc.
+- Only repositories directly referenced by the products in the moulin configuration *.yaml files are placed inside the **'xt-core'** entity. Those repositories are:
+  - xt-common - https://github.com/xen-troops/meta-xt-common
+  - xt-rcar - https://github.com/xen-troops/meta-xt-rcar
+  - xt-rpi5 - https://github.com/xen-troops/meta-xt-rpi5
+  - xt-vhost - https://github.com/xen-troops/meta-xt-vhost
+
+  This is necessary so that products can reference these repositories by their release tags - v1.0.0, v2.1.0, etc. For example:
+
+  - type: git
+  - url: https://github.com/xen-troops/meta-xt-common.git
+  - rev: v1.0.0
+
+- All other xen-troops repositories that get into the product through the above-mentioned **xt-core** repositories will be referenced in the Bitbake recipes using the commit IDs or the release tags ( or branches ) based on community-based naming. **This means no default usage of the Bitbake 'SRCREV = "${AUTOREV}"'!**
+
+  Such repositories are:
+    
+	- backends - https://github.com/xen-troops/snd_be, https://github.com/xen-troops/displ_be, https://github.com/xen-troops/camera_be
+    - xen - https://github.com/xen-troops/xen
+    - linux - https://github.com/xen-troops/linux
+    - optee-os - https://github.com/xen-troops/optee_os, https://github.com/xen-troops/optee_client, https://github.com/xen-troops/optee_test
+    - many others.

```

Comment:
I would recommend removing links in this block of text because we are using not only repos in `xen-troops` organization for linux, optee-os and some others. So, it is enough to list general names, without links,

Reply:
Hi @rshym, done.